src/Configuration/NodeDefinition/SslNodeDecorator.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\DbExtractorConfig\Configuration\NodeDefinition;
+
+use Symfony\Component\Config\Definition\Builder\NodeBuilder;
+
+class SslNodeDecorator
```

Comment:
Ono by to IMHO malo byt `SslNode` (ako som pisal skor), kedze vsade pouzivame taky styl skladania konfiguracie .... 


`TableNodesDecorator` je vynimka ... pretoze v `ConfigRowDefinition` musime tie kluce pridat k existujucej node,  a nie ako novu podpolozku:
https://github.com/keboola/db-extractor-config/blob/b8da6f36ba777050d67914ffcf6573bb0259e7b6/src/Configuration/ConfigRowDefinition.php#L56

------

V `ConfigDefinition` to zase pridavame ako podpolozku, preto je to `Decorator` aby bolo mozne spravit aj jedno aj druhe:
https://github.com/keboola/db-extractor-config/blob/b8da6f36ba777050d67914ffcf6573bb0259e7b6/src/Configuration/ConfigDefinition.php#L56-L57

----- 

Teda upravil by som nazov na `SslNode` a dedil od `ArrayNodeDefinition` .... inak ti iba staci vymenit `addNodes` za `init` a mas to, Vid `SshNode`, ... ide mi o to, aby ked sa niekto na to pozrie, tak sa necudoval a neskumal, preco je to spravene inak.
https://github.com/keboola/db-extractor-config/blob/COM-276-ondra-add-ssl-config/src/Configuration/NodeDefinition/SshNode.php